tests/import_proxy_test.go

```diff
@@ -90,6 +94,18 @@ var _ = Describe("Import Proxy tests", func() {
 		if utils.IsOpenshift(f.K8sClient) {
 			By("Reverting the cluster wide proxy spec to the original configuration")
 			cleanClusterWideProxy(ocpClient, clusterWideProxySpec)
+
+			By("Resuming MCPs")
+			manipulateMachineConfigPools(f, false)
+
+			By("Finish node reboots")
+			// Wait a bit for the reboot to start so we don't succeed immediately
+			time.Sleep(2 * time.Minute)
+			Eventually(func() error {
+				output, err := RunOcCommand(f, "wait", "mcp", "--all", "--for", "condition=updated", "--timeout", "1s")
+				By(output)
+				return err
+			}, 50*time.Minute, time.Second).ShouldNot(HaveOccurred())
```

Comment:
Just making sure this is seen, and keep in mind this will run for every test in the import proxy suite (potentially 5 hours).

Reply:
Right, but this will only be on openshift clusters.